jena-core/src/main/java/jena/RuleMap.java

```diff
@@ -50,7 +53,7 @@
  * </p>
  */
 public class RuleMap {
-    static { setLog4jConfiguration() ; }
+    static { setCmdLogging("jena-log4j.properties") ; }
     
```

Comment:
Can this be plain `setCmdLogging()` which (1) is sensitive to a local log4j.properties if the user that and (2) has a built-in config in case no properties file is found.